include/mesh/xdr_io.h

```diff
@@ -201,6 +201,15 @@ class XdrIO : public MeshInput<MeshBase>,
    */
   bool version_at_least_1_3_0() const;
 
+  /**
+   * \returns \p true if the current file has an XDR/XDA version that
+   * matches or exceeds 1.8.0.
+   *
+   * In this version we added support for writing the Mesh's "extra"
+   * integer fields to the file.
+   */
+  bool version_at_least_1_8_0() const;
```

Comment:
In hindsight we probably should have made this a single API with three int parameters, and some internal asserts to protect against typos.  Not complaining now, just noticing it's something we can consolidate later.

Reply:
I agree, sounds like a good refactoring to me. 

I saw a few other things that I would like to potentially try and refactor in `XdrIO` while working on this PR, for example in `write_serialized_nodes()` I simply continued the pattern of re-communicating the node ids for a (now) third time that was started when we added support for writing unique ids to the Xdr file. The node ids should probably just be saved on processor 0 in "chunks" the first time they are communicated, but I didn't want attempt to make that larger refactoring while also adding this new feature.